File: src/main/java/com/heybys/oddments/OddmentsApplication.java
package com.heybys.oddments;

import java.util.TimeZone;

import jakarta.annotation.PostConstruct;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@SpringBootApplication
public class OddmentsApplication {

    public static void main(String[] args) {
        SpringApplication.run(OddmentsApplication.class, args);
    }

    @PostConstruct
    public void init() {
        TimeZone.setDefault(TimeZone.getTimeZone("UTC"));
    }
}

File: src/main/kotlin/com/heybys/oddments/OddmentsApplication.kt
package com.heybys.oddments

import jakarta.annotation.PostConstruct
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import java.util.TimeZone

@SpringBootApplication
class OddmentsApplication {

    @PostConstruct
    fun init() {
        TimeZone.setDefault(TimeZone.getTimeZone("UTC"))
    }
}

fun main(args: Array<String>) {
    runApplication<OddmentsApplication>(*args)
}
